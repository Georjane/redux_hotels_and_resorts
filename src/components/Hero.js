import styled from 'styled-components';

const Hero = () => {
  const Button1 = styled.button`
  background: transparent;
  border-radius: 3px;
  border: 1px solid white;
  color: white;
  margin: 0.5em 1em;
  padding: 0.25em 1em;
`;
  return (
    <div>
      <div id="carouselExampleCaptions" className="carousel slide landingpage" data-ride="carousel">
        <ol className="carousel-indicators">
          <li data-target="#carouselExampleCaptions" data-slide-to="0" className="active" />
          <li data-target="#carouselExampleCaptions" data-slide-to="1" />
          <li data-target="#carouselExampleCaptions" data-slide-to="2" />
        </ol>
        <div className="carousel-inner">
          <div className="carousel-item active">
            <img src="images/img7.jpg" className="d-block w-100" alt="carosel" />
            <div className="carousel-caption d-none d-md-block">
              <h2>Leisure and Family Vacations</h2>
              <p>at Address Hotels + Resorts</p>
              <Button1>Learn More</Button1>
            </div>
          </div>
          <div className="carousel-item">
            <img src="images/img3.jpg" className="d-block w-100" alt="carosel" />
            <div className="carousel-caption d-none d-md-block">
              <h2>Fun-Filled Stays</h2>
              <p>at Address Hotels + Resorts</p>
              <Button1>Learn More</Button1>
            </div>
          </div>
          <div className="carousel-item">
            <img src="images/img5.jpg" className="d-block w-100" alt="carosel" />
            <div className="carousel-caption d-none d-md-block">
              <h2>Experience Expo 2021 Free!</h2>
              <p>Get 2 complimentary tickets to the worldâ€™s greatest show when you book with us.</p>
              <Button1>Learn More</Button1>
            </div>
          </div>
        </div>
        <a className="carousel-control-prev gradientover" href="#carouselExampleCaptions" role="button" data-slide="prev">
          <span className="carousel-control-prev-icon" aria-hidden="true" />
          <span className="sr-only">Previous</span>
        </a>
        <a className="carousel-control-next gradientover" href="#carouselExampleCaptions" role="button" data-slide="next">
          <span className="carousel-control-next-icon" aria-hidden="true" />
          <span className="sr-only">Next</span>
        </a>
      </div>
    </div>
  );
};
export default Hero;
