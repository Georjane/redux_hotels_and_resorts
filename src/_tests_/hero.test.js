import React from 'react';
import renderer from 'react-test-renderer';
import Hero from '../components/Hero';

it('renders the Hero properly', () => {
  const tree = renderer.create(<Hero />);
  expect(tree).toMatchSnapshot();
});
