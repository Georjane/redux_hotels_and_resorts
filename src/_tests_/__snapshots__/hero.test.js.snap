// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Hero properly 1`] = `
<div>
  <div
    className="carousel slide landingpage"
    data-ride="carousel"
    id="carouselExampleCaptions"
  >
    <ol
      className="carousel-indicators"
    >
      <li
        className="active"
        data-slide-to="0"
        data-target="#carouselExampleCaptions"
      />
      <li
        data-slide-to="1"
        data-target="#carouselExampleCaptions"
      />
      <li
        data-slide-to="2"
        data-target="#carouselExampleCaptions"
      />
    </ol>
    <div
      className="carousel-inner"
    >
      <div
        className="carousel-item active"
      >
        <img
          alt="carosel"
          className="d-block w-100"
          src="images/img7.jpg"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h2>
            Leisure and Family Vacations
          </h2>
          <p>
            at Address Hotels + Resorts
          </p>
          <button
            className="sc-bdfBQB bSgpNk"
          >
            Learn More
          </button>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="carosel"
          className="d-block w-100"
          src="images/img3.jpg"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h2>
            Fun-Filled Stays
          </h2>
          <p>
            at Address Hotels + Resorts
          </p>
          <button
            className="sc-bdfBQB bSgpNk"
          >
            Learn More
          </button>
        </div>
      </div>
      <div
        className="carousel-item"
      >
        <img
          alt="carosel"
          className="d-block w-100"
          src="images/img5.jpg"
        />
        <div
          className="carousel-caption d-none d-md-block"
        >
          <h2>
            Experience Expo 2021 Free!
          </h2>
          <p>
            Get 2 complimentary tickets to the worldâ€™s greatest show when you book with us.
          </p>
          <button
            className="sc-bdfBQB bSgpNk"
          >
            Learn More
          </button>
        </div>
      </div>
    </div>
    <a
      className="carousel-control-prev gradientover"
      data-slide="prev"
      href="#carouselExampleCaptions"
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-prev-icon"
      />
      <span
        className="sr-only"
      >
        Previous
      </span>
    </a>
    <a
      className="carousel-control-next gradientover"
      data-slide="next"
      href="#carouselExampleCaptions"
      role="button"
    >
      <span
        aria-hidden="true"
        className="carousel-control-next-icon"
      />
      <span
        className="sr-only"
      >
        Next
      </span>
    </a>
  </div>
</div>
`;
